torch/_thnn/utils.py

```diff
@@ -2,8 +2,11 @@
 import itertools
 import importlib
 
-THNN_H_PATH = os.path.join(os.path.dirname(__file__), '..', 'lib', 'THNN.h')
-THCUNN_H_PATH = os.path.join(os.path.dirname(__file__), '..', 'lib', 'THCUNN.h')
+try:
```

Comment:
this is the part that looks sketchy, worth a comment. otherwise PR looks good to go.